Makefile

```diff
@@ -166,12 +166,14 @@ endif
 	tinygo build -size short -o ./build/test.hex -target=circuitplay-express ./examples/lis2mdl/main.go
 	@md5sum ./build/test.hex
 	tinygo build -size short -o ./build/test.hex -target=feather-m0 ./examples/dht/main.go
+	@md5sum ./build/t
```

Comment:
Looks like the end of the line of text got cutoff here. That is why the tests are failing.

Reply:
next time i'll run them manually again, after resolving merge conflicts :)